### PR TITLE
fix: android slider seek时与viewPager 滑动冲突

### PR DESCRIPTION
**Pull request by huaihong1986**
Summary:
---------

Test Plan:
----------

### CODE REVIEW

**Comment by wojteg1337** [comment]
Hello, thanks for your contribution.

Please translate comments in code to English & fill the description of pull request.

**Comment by huaihong1986** [comment]
OK， I have done.

**Comment by wojteg1337** [comment]
Hi @huaihong1986,

The description of merge request is empty.

File: src/android/src/main/java/com/reactnativecommunity/slider/ReactSlider.java [hunk 1]
```diff
@@ -233,4 +234,23 @@ public void setThumbImage(final String uri) {
       setThumb(getThumb());
     }
   }
+    @Override
+    public boolean onTouchEvent(MotionEvent event) {
+        //设置父容器不拦截事件
+        if (getParent() != null) {
+            try {
+                getParent().requestDisallowInterceptTouchEvent(true);
+                //SeekBar 放在其他视图中，手指放在 SeekBar 上面不会立即触发滑动操作
```
Comment (BartoszKlonowski) [inline src/android/src/main/java/com/reactnativecommunity/slider/ReactSlider.java:243]: The code should always be self-explanatory.
If comments are required for the code to be more understandable they should be written in English.

Applies also to line 239.

File: src/android/src/main/java/com/reactnativecommunity/slider/ReactSlider.java [hunk 2]
```diff
@@ -233,4 +234,23 @@ public void setThumbImage(final String uri) {
...
+                if (event.getAction() == MotionEvent.ACTION_DOWN) {
+                    MotionEvent evup = MotionEvent.obtain(event);
+                    evup.setAction(MotionEvent.ACTION_MOVE);
+                    dispatchTouchEvent(evup);
+                    evup.recycle();
+                }
+            } catch (Exception exception) {
```
Comment (BartoszKlonowski) [inline src/android/src/main/java/com/reactnativecommunity/slider/ReactSlider.java:250]: Why leaving an empty exception here?

File: src/android/src/main/java/com/reactnativecommunity/slider/ReactSlider.java [hunk 3]
```diff
@@ -233,4 +234,23 @@ public void setThumbImage(final String uri) {
       setThumb(getThumb());
     }
   }
+    @Override
+    public boolean onTouchEvent(MotionEvent event) {
+        //设置父容器不拦截事件
+        if (getParent() != null) {
+            try {
```
Comment (BartoszKlonowski) [inline src/android/src/main/java/com/reactnativecommunity/slider/ReactSlider.java:241]: What exceptions do we expect here to be thrown by `MotionEvent`?
`getParent()` is already checked against `null`.

**Pull request closed without merging**